shared/ui-components/src/authentication/api_requests.ts

```diff
@@ -10,7 +10,7 @@ export const renewToken = async (client: AxiosInstance) => {
 
 export const login = async (apiBase: string, email: string, password: string) => {
   const res = await axios.post<{ accessToken: string }>(`${apiBase}/auth/login`, {
-    username: email,
+    email: email,
```

Comment:
My only concern here is that I don't think we're ready to say that username == email address across the board, since a small but important part of our user population seems not to have email addresses. I realize that isn't what this PR is about, so we can handle it elsewhere if you think that's more appropriate, but I just wanted to make sure we weren't taking a step in the wrong direction.

(I do think it's 100% fine that the test username is an email address, since I think that's the normal case and we want to encourage it where appropriate. I just want to make sure we're not setting up a model where engineers assume that's the only option.)

Reply:
Ok, split this off to #461